doc/requirements.txt

```diff
@@ -1,4 +1,3 @@
-docutils<0.18
```

Comment:
Why the removal?

Reply:
https://github.com/readthedocs/sphinx_rtd_theme/issues/1323 - this is no longer needed when we upgrade to the later version